### DISCUSSION TITLE
Import a react component in a content script

### DISCUSSION THREAD

**Discussion by blemmmm** [crxjs/chrome-extension-tools#685] [Q&A]
i have this code in my content script

![image](https://user-images.githubusercontent.com/70497919/229765664-d7a531dd-50ab-4ad1-be7d-378de9040e9a.png)

but I have this error in crxjs

![image](https://user-images.githubusercontent.com/70497919/229765043-ef9354bd-921d-4498-8018-907b356d7375.png)

how to correctly import a component?

**Comment by Toumash** [comment]
See this example repo: https://github.com/crxjs/vite-4-react/tree/main

Is it a .tsx/.jsx file as in the sample?
